core/src/main/java/org/geysermc/geyser/network/GameProtocol.java

```diff
@@ -184,7 +174,7 @@ public static int getJavaProtocolVersion() {
      * @return the supported Minecraft: Java Edition version
      */
     public static String getJavaMinecraftVersion() {
-        return "1.21.10";
+        return "1.21.11";
```

Comment:
This can probably return to being `DEFAULT_JAVA_CODEC.getMinecraftVersion();` for the time being.